test/fw/ptl/utils/pbs_logutils.py

```diff
@@ -358,19 +368,26 @@ def match_msg(self, lines, msg, allmatch=False, regexp=False,
                           specified time
         :param endtime: If set ignore matches that occur after
                         specified time
+         :param syslog: If True, checks for syslog format of date.
+                       Defaults to False.
+        :type str
         """
         linecount = 0
         ret = []
         if lines:
             for l in lines:
+                if syslog:
+                    dt_str = l[:15]
```

Comment:
Since in case of syslog we do not support microsecond logging, should we just convert starttime and endtime to int so that we are sure we are not using a float number?